packages/plugin-ext/src/plugin/type-converters.spec.ts

```diff
@@ -287,6 +300,14 @@ describe('Type converters:', () => {
             assert.deepStrictEqual(result, shellTaskDto);
         });
 
+        it('shell task with ShellStringQuoted command should convert to task dto', () => {
```

Comment:
minor: similarly to above:

```suggestion
        it('should convert to task dto with `ShellStringQuoted` command', () => {
```